i3pystatus/timewarrior.py

```diff
@@ -0,0 +1,77 @@
+from i3pystatus import IntervalModule
+from json import loads
+from dateutil.parser import parse
+from dateutil.relativedelta import relativedelta
+from datetime import datetime, timezone
+import subprocess
+
+
+class TImewarrior(IntervalModule):
+    """
+    Show current Timewarrior tracking
+    Requires `json` `dateutil`
+
+    Formaters:
+
+    * `{tags}`  — contains tags of current track
+    * `{start}` - contains start of track
+    * `{duration}` — contains time of current track
+    """
+
+    format = '{duration}'
+    duration_format = '{years}y{months}m{days}d{hours}h{minutes}m{seconds}s'
+    enable_stop = True
+    enable_continue = True
+    color_running = '#00FF00'
+    color_stopped = '#F00000'
+    on_rightclick = 'stop_or_continue'
+    track = None
+
+    settings = (
+        ('format', 'format string'),
+        ('enable_stop', 'Allow right click to stop tracking'),
+        ('enable_continue', 'ALlow right click to continue tracking'),
+        ('color_running', '#00FF00'),
+        ('color_stopped', '#F00000'),
+    )
+
+    def loadTrack(self):
+        try:
+            tracks_json = subprocess.check_output(['timew', 'export'])
+            tracks = loads(tracks_json.decode("utf-8"))
+            self.track = tracks[-1]
+
+        except ValueError:
+            print('Decoding JSON has failed')
```

Comment:
Should rethrow an exception here, that way the error will be visible in the status bar and not break things.